### PR TITLE
IGVF-511-remove-yeast-wholeorganism

### PR DESCRIPTION
**Pull request by corismall**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/biosample.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`biosample.json`*
 
+
+### Release v10
+
+* Remove `saccharomyces`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/biosample.md:6]: I think this should be "Remove taxa saccharomyces."

File: src/igvfd/schemas/changelogs/in_vitro_system.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`in_vitro_system.json`*
 
+
+### Schema version 7
+
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/in_vitro_system.md:5]: I think this should be "Remove taxa saccharomyces."

File: src/igvfd/schemas/changelogs/primary_cell.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`primary_cell.json`*
 
+
+### Schema version 11
+
+* Remove `saccharomyces`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/primary_cell.md:6]: I think this should be "Remove taxa saccharomyces."

File: src/igvfd/tests/test_schema_tissue.py [hunk 4]
```diff
@@ -22,14 +22,6 @@ def test_age_unit_dependency(tissue, testapp):
         tissue['@id'],
         {'age_units': 'minute'})
     assert res.status_code == 200
-    res = testapp.patch_json(
```
Comment (gabdank) [inline src/igvfd/tests/test_schema_tissue.py:25]: I think you can just revert this test into 422?

File: src/igvfd/tests/test_schema_whole_organism.py [hunk 5]
```diff
@@ -32,10 +32,6 @@ def test_taxa_dependency(whole_organism, testapp):
         whole_organism['@id'],
         {'taxa': 'Mus musculus'})
     assert res.status_code == 200
-    res = testapp.patch_json(
```
Comment (gabdank) [inline src/igvfd/tests/test_schema_whole_organism.py:35]: I think you can revert the test into 422?

File: src/igvfd/tests/test_upgrade_whole_organism.py [hunk 6]
```diff
@@ -66,3 +66,9 @@ def test_whole_organism_upgrade_8_9(upgrader, whole_organism_v8):
     value = upgrader.upgrade('whole_organism', whole_organism_v8, current_version='8', target_version='9')
     assert value['schema_version'] == '9'
     assert value['sorted_fraction_detail'] == 'Default upgrade text: please add more details about sorted_fraction, see sample.json for description.'
+
+
+def test_whole_organism_upgrade_9_10(upgrader, whole_organism_v9):
+    value = upgrader.upgrade('whole_organism', whole_organism_v9, current_version='9', target_version='10')
+    assert value['schema_version'] == '10'
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_whole_organism.py:73]: the notes should be checked as well?

**Review by gabdank — CHANGES_REQUESTED** [review]
see comments

File: src/igvfd/upgrade/sample.py [hunk 7]
```diff
@@ -143,3 +143,14 @@ def sample_10_11(value, system):
...
+
+
+@upgrade_step('primary_cell', '10', '11')
+@upgrade_step('tissue', '10', '11')
+@upgrade_step('whole_organism', '9', '10')
+@upgrade_step('in_vitro_system', '6', '7')
+def biosample_6_7(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-511
+    if value['taxa'] == 'Saccharomyces':
+        value['notes'] = 'Previous taxa: ' + value['taxa'] + ' is no longer valid.'
```
Comment (iyoungworth) [inline src/igvfd/upgrade/sample.py:155]: This line will overwrite any previous value for 'notes' in each object, so it's best to check for the current value and instead append your new comment. There are multiple ways to do this, you can take a look at some examples in /upgrade/treatment.py and /upgrade/donor.py to use as a starting point

**Review by iyoungworth — CHANGES_REQUESTED** [review]
The changelogs and tests look good, but one change in the upgrade code is requested. Also, the schema_version default value in each schema.json file (just the concrete class objects) needs to be bumped up by one

**Review by iyoungworth — APPROVED** [review]
Code all looks good, pass. (Schema_version defaults are bumped by 2 to account for a previous merged upgrade ticket that missed this step)

**Review by iyoungworth — DISMISSED** [review]

**Review by iyoungworth — APPROVED** [review]

**Pull request merged**